packages/sommelier.rb

```diff
@@ -21,8 +21,7 @@ class Sommelier < Package
        i686: 'f00ace87c154fa1c4f786656db369004612b51c27236dbdca9c4f0fd230496eb',
      x86_64: '29b161ee80fbc1f334bfaf7d191ce8b37c0b7cff414d3e639934b10de64280d7'
   })
```

Comment:
We will need new binaries so remove and bump the version?

Reply:
@supechicken Can you confirm that this sommelier works when built?

Reply:
@supechicken: What is the status of this PR?

Reply:
I need to get back my laptop from my friend for testing

Reply:
> We will need new binaries so remove and bump the version?

Done